lib/class-wp-theme-json-gutenberg.php

```diff
@@ -397,8 +407,10 @@ class WP_Theme_JSON_Gutenberg {
 		),
 		'custom'                        => null,
 		'dimensions'                    => array(
-			'aspectRatio' => null,
-			'minHeight'   => null,
+			'aspectRatio'         => null,
```

Comment:
Unrelated: I noticed that you can't disable aspect ration for Image block using this setting. cc @draganescu

Reply:
The initial `null` value is how theme json defines flag settings, and it's not important at the moment.

Setting it to `false` disables the aspect ratio UI for blocks. You can test it with the Cover block. I think behavior should be consistent across the blocks that use aspect ratio control.

cc @andrewserong

Reply:
Yes, indeed setting to false hides this dimension tool from the cover block but it does not hide it from the image block. The reason is that the cover block uses the supports while the image block plops the control in the inspector.

Reply:
> Yes. I'm aware of that.

I knew that, you told me where to look! 😂  I mentioned here for archaeology reasons.

Reply:
Created a follow up issue for the aspect ratio block support cleanup: https://github.com/WordPress/gutenberg/issues/61432